Automattic/wp-calypso
# Inline Help: Add an onboarding welcome at the preview page.

## Changes proposed in this Pull Request

<img width="415" alt="screen shot 2019-01-29 at 4 15 51 pm" src="https://user-images.githubusercontent.com/6458278/51885784-32e51b00-23e1-11e9-976c-9f25fe325ea9.png">

Testing out whether we can/should open the inline popup with onboarding welcome content via a query param.

### Considerations ❓ 
The query param `welcome` triggers the content. Should we program in some limitation to this, for example, the onboarding inline-help content should only show, despite the presence of the `welcome` parameter when:
1. All checklist items **have not** been completed
2. The user has created the current site in the last minute, or has just been redirected from the onboarding flow

~Also, the design omits regular help content (Help/contact links and search) from the bubble. Should we include something to indicate to the user that help is available, to ensure that they know the intended purpose of inline help?~ We've also added a Help link to the footer.

## Testing instructions

1. Create a new site
2. When you're redirected to preview, check that the URL contains the `?welcome` parameter
http://calypso.localhost:3000/view/yoursite.wordpress.com?welcome The inline-help with welcome message should be present
3. Click **Contact us**, then click **Back**. The inline-help with welcome message should be present
3. Close by clicking **Not now**, then reopen the inline help bubble. The regular help content (Help link, search etc) should show.
4. Refresh the page to show the congrats message again, and click **< Help**. The bubble content should display the regular help content plus checklist progress widget.
5. Refresh the page to show the congrats message again, and click **Start Customizing**. You should be redirected to the checklist, and reopening the inline help bubble should display regular help content .
